hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml

```diff
@@ -0,0 +1,8 @@
+name: tfkeras mobilenet_v2
+description: an image with the encoder of mobilenet.v2 from tfkeras
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://jina.ai
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.0
+vendor: Jina AI Limited
+license: apache-2.0
```

Comment:
missing `update` key. Would you please have a look at [`update`](https://github.com/jina-ai/jina-hub#schema-of-manifestyml) in the documentation?